src/components/FeeModal/FeeBreakdown.tsx

```diff
@@ -24,7 +25,44 @@ export const FeeBreakdown = ({ feeModel, inputAmountUsd }: FeeBreakdownProps) =>
     feeUsd: affiliateFeeAmountUsd,
     foxDiscountUsd,
     feeUsdBeforeDiscount,
+    feeBps,
   } = useAppSelector(state => selectCalculatedFees(state, { feeModel, inputAmountUsd }))
+  const appliedDiscountType = useAppSelector(state =>
+    selectAppliedDiscountType(state, { feeModel, inputAmountUsd }),
+  )
+  const greenColor = useColorModeValue('green.500', 'green.300')
+
+  const discountTypeTranslation = useMemo(() => {
+    return translate(`foxDiscounts.${appliedDiscountType}`)
+  }, [appliedDiscountType, translate])
+
+  const discountLabelTranslation = useMemo(() => {
+    return translate(`foxDiscounts.discountLabel`, { discountType: discountTypeTranslation })
+  }, [discountTypeTranslation, translate])
+
+  const totalAmount = useMemo(() => {
+    if (feeBps.isZero())
+      return (
+        <Row.Value fontSize='lg'>
+          <Text translation='trade.free' fontWeight='semibold' color={greenColor} />
+        </Row.Value>
+      )
+
+    return (
+      <Row.Value fontSize='lg'>
+        <Amount.Fiat
+          fiatType='USD'
+          value={affiliateFeeAmountUsd.toFixed(2, BigNumber.ROUND_HALF_UP)}
+        />
+      </Row.Value>
+    )
+  }, [affiliateFeeAmountUsd, feeBps, greenColor])
+
+  const totalDiscount = useMemo(() => {
+    if (feeBps.isZero()) return feeUsdBeforeDiscount.toFixed(2, BigNumber.ROUND_HALF_UP)
```

Comment:
see https://github.com/shapeshift/web/pull/8896/files#r1963284577

using `feeUsdBeforeDiscount` as foxDiscountUsd is taking in account very very small fees (which is not a bug but something we don't consume at interface level for now as we apply discount starting from 1 bps, and not 0.1 bps), to reflect 100% discount in the interface while the next instruction will use the real discount